### PR TITLE
Removed Cluster and Host divs that are not required

### PR DESCRIPTION
**Pull request by a2batic**
Remove following things:-
* Donut-cart
  - Component from ```src/commons/components```
  - import statement from ```src/commons/scss/main.scss```
* Divs from ```src/modules/cluster/cluster-list/cluster-list.html```
  - IOPSData
  -  Hosts
  -  Pools or Volumes
  -  Alerts
* Functions from ```src/modules/cluster/cluster-list.js```
  -  clusterIopsTrend
  -  generateIopsTrend
  - setupXData
  -  setupYData
* Divs from ```src/commons/modules/host/host-list/host-list.html```
  - Comment for 'Add' Button
  - Donut chart for storage
  - Donut chart for CPU time
  - Donut chart for memory
  - Alerts

The Cluster list looks like following:-

screenshot from 2017-07-28 11-10-46

The Host list looks like following:-

screenshot from 2017-07-28 11-11-16

### CODE REVIEW

**Comment by a2batic** [comment]
@gnehapk, @cloudbehl , Please review

**Review by gnehapk — APPROVED** [review]

**Comment by cloudbehl** [comment]
@a2batic can you create a Issue for the commit and add it to the commit message.

**Comment by cloudbehl** [comment]
@a2batic Please update the PR with the following changes -
1) Please update the host-list.js with the changes you have made in host-list.html
2) Remove kabab menu icons from a cluster and a host list page.

**Comment by a2batic** [comment]
@cloudbehl, acknowledge, will do the changes.

File: src/modules/host/host-list/host-list.js [hunk 1]
```diff
@@ -73,16 +73,6 @@
                 host.status = list[i].status;
                 host.name = list[i].fqdn;
                 host.role = nodeStore.findRole(list[i].tags);
-                host.cluster_name = list[i].tendrlcontext.cluster_name;
```
Comment (gnehapk) [inline src/modules/host/host-list/host-list.js]: @a2batic HTML contains the cluster Name column, then why you have removed cluster_name property in JS file?

Comment (a2batic) [reply src/modules/host/host-list/host-list.js]: @gnehapk, I removed it by mistake, I will update the PR

**Review by gnehapk — APPROVED** [review]

**Pull request merged**